system_metrics_collector/include/moving_average_statistics/moving_average.hpp

```diff
@@ -0,0 +1,189 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This header is in include/ do we expect this class to be used from the outside?

Reply:
At some point we will want to do so, but moving to src for now while we develop.

Reply:
Moved to src